### DISCUSSION TITLE
hypr and waybar dots not more available

### DISCUSSION THREAD

**Discussion by cashmeredev** [1amSimp1e/dots#51] [General]
Hey I reinstalled my laptop and wanted to use your dots again :) but unfortunately i couldn't find your hypr and waybar dots anymore. Is it possible to reupload them again?

**Comment by 1amSimp1e** [comment]
did you read the README.md ?

**Reply by cashmeredev** [reply]
I didn't clicked on the themes my bad. Already found it
